sdcm/nemesis.py

```diff
@@ -2191,10 +2191,7 @@ def disable_disrupt_corrupt_then_scrub(self):
 
         self.log.debug("Rebuild sstable file by scrub, corrupted data file will be skipped.")
         for ks_cf in self.cluster.get_non_system_ks_cf_list(self.target_node):
-            scrub_cmd = 'curl -s -X GET --header "Content-Type: application/json" --header ' \
-                        '"Accept: application/json" "http://127.0.0.1:10000/storage_service/keyspace_scrub/{}?skip_corrupted=true"'.format(
-                            ks_cf.split('.')[0])
-            self.target_node.remoter.run(scrub_cmd)
+            self.target_node.remoter.nodetool(f"scrub {ks_cf.split('.')[0]}")
```

Comment:
@amoskong since when do we have `nodetool` method on remoter?

Reply:
Fixed.